controller/rancher/rancher.go

```diff
@@ -156,23 +183,46 @@ func (lbc *LoadBalancerController) BuildConfigFromMetadata(lbName, envUUID, self
 		}
 	}
 	frontendsMap := map[string]*config.FrontendService{}
-	// fetch certificates
+
```

Comment:
It would also help us in the future to move on from monolithic CertificateFetcher to the pluggable ones (one would read from metadata, another from the directory, another from Vault service, etc)